backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/settings/ModularContractSettings.java

```diff
@@ -118,6 +120,17 @@ public List<ModuleConfig> getModuleConfigs(@NonNull final ComputingMethodType co
 				.toList();
 	}
 
+	@NonNull
+	public ModuleConfig getModuleConfigByIdOrError(@NonNull final ModuleConfigAndSettingsId moduleConfigAndSettingsId)
```

Comment:
minor: imho "OrError" suffix is not needed because if u are returning ModuleConfig instead of Optional,
to me, it's pretty obvious that an exception is thrown in case the config was not found.